<ui version="4.0" >
 <class>PickNameDialogBase</class>
 <widget class="QDialog" name="PickNameDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Pick name from list...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="TextLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Pick one name from the suggestion list!</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignTop</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox" >
         <property name="toolTip" >
          <string>Tell what to do with your selection</string>
         </property>
         <item>
          <property name="text" >
           <string>Replace current dataset/group name</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Append to existing dataset/group name</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QPushButton" name="PushButtonOk" >
         <property name="maximumSize" >
          <size>
           <width>59</width>
           <height>25</height>
          </size>
         </property>
         <property name="text" >
          <string>OK</string>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PushButtonCancel" >
         <property name="maximumSize" >
          <size>
           <width>59</width>
           <height>25</height>
          </size>
         </property>
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Suggestion list</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QTreeWidget" name="treeWidgetSuggest" >
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="sortingEnabled" >
         <bool>true</bool>
        </property>
        <property name="allColumnsShowFocus" >
         <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Info type</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Suggested string</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
